### DISCUSSION TITLE
请问使用CSV汉化后如何还原成原本的语言

### DISCUSSION THREAD

**Discussion by YangGlen** [GpointChen/FFXIVChnTextPatch-GP#100] [Q&A] [answered]
你好，我使用CSV汉化后想要还原成原本的日文遇到了问题
按下还原后(不论是否在设置那选择资料夹)重新登入游戏，游戏中的语言还是汉化后的版本
请问该如何操作才能正常还原呢?

**Comment by qwe3410475** [comment, marked as answer]
你可能點了兩次漢化 導致程式幫你備份的文件又被替換成已經漢化的文件
請不要點兩次漢化 會覆蓋幫你備份的文件
只能請你重新下載 或 找朋友給你那6個文件

**Comment by YangGlen** [comment]
感谢你的回答，请问是哪6个檔案呢?

**Comment by kalispp** [comment]
C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv
000000.win32.dat0
000000.win32.index
000000.win32.index2
0a0000.win32.dat0
0a0000.win32.index
0a0000.win32.index2
